src/main/java/com/raoulvdberge/refinedstorage/tile/TileController.java

```diff
@@ -105,18 +108,17 @@ public Integer getValue(TileController tile) {
 
             for (INetworkNode node : tile.nodeGraph.all()) {
                 if (node.canUpdate()) {
-                    IBlockState state = tile.getWorld().getBlockState(node.getPosition());
+                    ItemStack itemStack = node.getItemStack();
+                    if (itemStack == null) {
```

Comment:
That's why I'm saying it. I think that you should return Air ItemStack with consistency with 1.11